graph-builder/src/graph.rs

```diff
@@ -62,9 +64,10 @@ lazy_static! {
         vec![5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 15.0, 20.0, 30.0 ]
     ))
     .unwrap();
-    static ref V1_GRAPH_INCOMING_REQS: Counter = Counter::new(
-        "v1_graph_incoming_requests_total",
-        "Total number of incoming HTTP client request to /graph"
+    static ref GRAPH_INCOMING_REQS: IntCounterVec = IntCounterVec::new(
+        Opts::new("graph_incoming_requests_total",
+        "Total number of incoming HTTP client request to /graph"),
```

Comment:
Drop `to /graph`?  And expect folks to notice the `endpoint` label to understand the actual target.